roles/dnsmasq/files/etc/dnsmasq.d/10-dnsmasq

```diff
@@ -0,0 +1,11 @@
+# Performance tweaks
+domain-needed
+bogus-priv
+dns-forward-max=2000
+cache-size=1000
+
+# use AWS servers for recursive lookups
+server=/#/%%DHCP_DNS%%
+
+# Use coredns for tag_namespace lookups
+#server=/%%TAG_NAMESPACE%%./%%COREDNS%%
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, this would remove the comment https://github.com/nxtlytics/ivy-ami-bakery/blob/37b568283d540ad42a72390f710930799c8f05c6/roles/dnsmasq/files/opt/ivy/set-dnsmasq-dhcp.sh#L45-L49